pkg/utils/slice/slice.go

```diff
@@ -0,0 +1,12 @@
+package slice
+
+// Contains checks if a slice contains an element
+func Contains(list []string, e string) bool {
+	for _, x := range list {
+		if x == e {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
This can be addressed later, but I think we shouldn't have this utility function at all. This is a simple `for` loop that works only for strings and we have not been using this in other parts of the code.